pack/tdc/tdcc.json

```diff
@@ -301,6 +301,178 @@
         "text": "Easy / Standard\n[skull]: -X. X is the current Disturbance (to a maximum of 6).\n[cultist]: -4. If you fail, draw the top card of the encounter deck. If you fail by 3 or more, that card gains peril and surge.\n[tablet]: -4. For each point you fail by, you must either take 1 horror or discard 1 card from your hand.\n[elder_thing]: -3. Reveal another token.",
         "type_code": "scenario"
     },
+    {
+        "back_link": "11674b",
+        "code": "11674",
+        "doom": null,
+        "double_sided": true,
```

Comment:
When a card has a `back_link`, it should not have `double_sided` (causes rendering error in ArkhamDB).

```suggestion
```